kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -3,8 +3,8 @@
 ERR_APT_INSTALL_TIMEOUT=9 # Timeout installing required apt packages
 ERR_AKSE_DOWNLOAD=10 # Failure downloading AKS-Engine binaries
 ERR_AKSE_DEPLOY=12 # Failure calling AKS-Engine's deploy operation
-ERR_TEMPLATE_DOWNLOAD=13 # Failure downloading AKS-Engine template
-ERR_INVALID_AGENT_COUNT_VALUE=14 # Both Windows and Linux agent value is zero 
+ERR_TEMPLATE_GENERATION=13 # Failure downloading AKS-Engine template
```

Comment:
Nit: Update the comment to reflect the new behaviour. Also you need to update https://github.com/msazurestackworkloads/azurestack-gallery/blob/master/kubernetes/docs/troubleshooting.md

Reply:
I would create a new error code to avoid potential confusion